tests/yam/agama/agama.pm

```diff
@@ -26,6 +29,15 @@ sub run {
     my $test = get_required_var('AGAMA_TEST');
     my $reboot_page = $testapi::distri->get_reboot();
 
+    if (is_sle() && is_s390x()) {
+        # The ISO does not have SUSE ca-certs, so we need to install them manually.
+        # Record a soft failure for this, when the bug is opened.
```

Comment:
you can drop this comment line, the bug is already opened.

Reply:
9f6fb3b18ccbfb0753821bb54e5b3b78455a4e7e